File: StreamusicClientAndroid/LogicaDeNegocio/Clases/Metadatos.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Logica.Clases
{
    public class Metadatos
    {
        public string Id { get; set; }
        public string IdCancion { get; set; }
        public string IdConsumidor { get; set; }
        public double CantidadDeVecesEscuchada { get; set; }
        public DateTime FechaPrimeraEscucha { get; set; }
        public DateTime FechaUltimaEscucha { get; set; }
        public int Calificacion { get; set; }
        public bool MeGusta { get; set; }
    }
}

File: Microservicios/MS_Metadatos/src/models/Metadatos.js
'use strict'
const mongoose = require('../config/mongoose');

const Schema = mongoose.Schema;

var MetadatoSchema = new Schema({ 

    IdCancion: {type: String, required: true},
    IdConsumidor: {type: String, required: true},
    CantidadDeVecesEscuchada: {type: Number, required: true},
    FechaPrimeraEscucha: { type: Date, default: Date.now },
    FechaUltimaEscucha: { type: Date, default: Date.now },
    Calificacion: { type: Number, required: true, max: 5, min: 0 },
    MeGusta: { type: Boolean, default: false}

}, {collection: 'Metadatos' });

const model = mongoose.model('Metadatos', MetadatoSchema);
module.exports = model;